hooks/server.js

```diff
@@ -42,5 +42,10 @@ module.exports = function Server(cb) {
         }
     });
 
+    // TODO discuss placement
```

Comment:
I don't want to make any assumptions as to what middleware/plugins are enabled by default. I've gone ahead and made it easier to enable some of the more common restify middleware plugins, but as far as enabling them by default, i think we should leave that up to individual apps to decide.. agree?